src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -4,10 +4,11 @@
     <TargetFrameworks>netcoreapp3.1;netstandard2.0;net462</TargetFrameworks>
     <AssemblyName>Microsoft.Azure.WebJobs.Extensions.DurableTask</AssemblyName>
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.DurableTask</RootNamespace>
-    <MajorVersion>2</MajorVersion>
-    <MinorVersion>11</MinorVersion>
-    <PatchVersion>1</PatchVersion>
-    <Version>$(MajorVersion).$(MinorVersion).$(PatchVersion)</Version>
+    <MajorVersion>3</MajorVersion>
+    <MinorVersion>0</MinorVersion>
+    <PatchVersion>0</PatchVersion>
+    <VersionSuffix>preview.2</VersionSuffix>
+    <Version>$(MajorVersion).$(MinorVersion).$(PatchVersion)-$(VersionSuffix)</Version>
```

Comment:
nit: this will break when we decide to remove the version suffix. Consider factoring this differently so that it works whether or not we have a version suffix for the nuget package.

Reply:
Makes sense. I can actually just use the built-in `<VersionPrefix>` property instead and rely on the default construction of `<Version>` based on `<VersionPrefix>` and `<VersionSuffix>`